spec/deck_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/card'
+require './lib/guess'
+require './lib/deck'
+require 'rspec'
+
+RSpec.describe Deck do
+  it "exists" do
+    deck = Deck.new
```

Comment:
This is the pattern that we are trying to help you get away from. Where in the interaction pattern is a deck ever created like this? Even in your exists test, include ALL of the set up that a deck needs.